daffodil-test/src/test/resources/org/apache/daffodil/section05/simple_types/SimpleTypes.tdml

```diff
@@ -3683,6 +3683,18 @@
     </tdml:errors>
   </tdml:unparserTestCase>
 
+  <tdml:parserTestCase name="double_negativeZero" root="doubleText"
+    model="SimpleTypes-Embedded.dfdl.xsd" description="Section 5 Schema types-double - DFDL-5-009R"
```

Comment:
(Aside) I am interested in ways to test Daffodil's C code generator more thoroughly.  However, the majority of Daffodil's tests (like this one) parse textual data, not binary data.  Even when I find binary data tests, they might also use hidden groups, input and/or output value calculations, variables, or other unsupported features.  Is there an easy way to identify existing Daffodil tests which could be run with "daffodil test -I daffodilC" successfully (besides just trying it on all tests and seeing which ones pass)?

Reply:
I suggest trying all tests and see what ones pass :-)